runtime/builtin_types_test.go

```diff
@@ -425,6 +425,69 @@ func TestBuiltinSetAttr(t *testing.T) {
 	}
 }
 
+func TestRawInput(t *testing.T) {
+	inputTest := func(s string, cases []invokeTestCase) {
```

Comment:
This organization seems like a reasonable way to support an additional parameter, but it's a little unorthodox. The way I've usually done it is something like this:

```
fun := wrapFuncForTest(func(f *Frame, s string, args ...*Object) (*Object, *BaseException) {
  ...
  go func() {
    w.Write([]byte(s))
    w.Close()
  }()
  ...
    in, raised := builtinRawInput(f, args, nil)
  ...
})
...
    {args: wrapArgs("hello\n", "show"), want: newTestTuple("hello", "show").ToObject()},
```